typescript/src/ethereum.ts

```diff
@@ -392,7 +515,7 @@ export class Bridge implements ChainBridge {
    * @see {ChainBridge#activeWalletPublicKey}
    */
   async activeWalletPublicKey(): Promise<string | undefined> {
-    const activeWalletPubKeyHash = await this._bridge.activeWalletPubKeyHash()
+    const activeWalletPubKeyHash = await this._instance.activeWalletPubKeyHash()
```

Comment:
Just ponting out that we decided to ditch `pubKeyHash` abbreviation and use full `publicKeyHash` name. We will use it everywhere in tbtc-v2.ts lib, but all things related to the contract, like methods or events should stay the same and consist `pubKeyHash` naming. You will probably notice that when you start resolving conflicts with the main branch. For example in this case it would be:

```suggestion
    const activeWalletPublicKeyHash = await this._instance.activeWalletPubKeyHash()
```

For more info you can see the discussion here: https://github.com/keep-network/tbtc-v2/pull/441#discussion_r1060021534